src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
-'use strict';
+const allItems = [...document.querySelectorAll('span')];
 
-// write your code here
+const populations = allItems.filter(
+  (element) => element.className === 'population',
+);
+const total = allItems.filter(
+  (element) => element.className === 'total-population',
+)[0];
+const average = allItems.filter(
+  (element) => element.className === 'average-population',
+)[0];
```

Comment:
you can use these classnames to get proper selector instead of filtering all span elements